modules/oadp-restic-issues.adoc

```diff
@@ -23,14 +23,15 @@ You can resolve this issue by creating a supplemental group for `Restic` and add
 
 . Create a supplemental group for `Restic` on the NFS data volume.
 . Set the `setgid` bit on the NFS directories so that group ownership is inherited.
-. Add the `spec.configuration.restic.supplementalGroups` parameter and the group ID to the `DataProtectionApplication` manifest, as in the following example:
```

Comment:
I would use "as the following example shows" or "as shown in the following example".

Reply:
Done. thanks.